Upgrade Google APIs Version

### Checked other resources

- [X] I added a very descriptive title to this issue.
- [X] I searched the LangChain.js documentation with the integrated search.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I am sure that this is a bug in LangChain.js rather than my code.
- [X] The bug is not resolved by updating to the latest stable version of LangChain (or the specific integration package).

### Example Code

### Steps to Reproduce:
1. Set up a project with googleapis@^140.0.0 as a dependency.
2. Add @langchain/community@^0.2.27 as a dependency.
3. Run npm install.
4. Observe the dependency conflict error.

### Error Message and Stack Trace (if applicable)

### Detailed Description:
The specific error encountered is as follows:
npm ERR! While resolving: @langchain/community@0.2.27
npm ERR! Found: googleapis@140.0.0
npm ERR! node_modules/googleapis
npm ERR!   googleapis@"^140.0.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peerOptional googleapis@"^126.0.1" from @langchain/community@0.2.27
npm ERR! node_modules/@langchain/community
npm ERR!   peerOptional @langchain/community@"*" from langchain@0.2.16
npm ERR!   node_modules/@langchain/community/node_modules/langchain
npm ERR!     langchain@"~0.2.3" from @langchain/community@0.2.27
npm ERR!   @langchain/community@"^0.2.27" from the root project



### Description

We are encountering dependency conflicts due to the current version of googleapis used in our project. Our project requires googleapis@^140.0.0, but the @langchain/community package specifies a peer dependency on googleapis@^126.0.1. This mismatch is causing npm to fail in resolving the dependencies, leading to installation errors.

### Proposed Solution:
We propose upgrading the googleapis dependency in the @langchain/community package to ^140.0.0 to align with the version required. This upgrade will help resolve the dependency conflict and allow for smoother installation and integration.

### Impact:
This issue is blocking project setup and development, as we cannot proceed with the current dependency conflict. Resolving this will allow us to continue development without resorting to workaround options like --legacy-peer-deps, --force, or resolutions.

### Additional Context:
We have attempted various solutions, including using the overrides option in package.json, but the conflict persists. Upgrading the googleapis version in the @langchain/community package seems to be the most viable solution.


### System Info

node - v20.10.0
npm - v10.2.3
os- windows 10 pro

--- a/libs/langchain-community/package.json
+++ b/libs/langchain-community/package.json
@@ -293,7 +293,7 @@
     "faiss-node": "^0.5.1",
     "firebase-admin": "^11.9.0 || ^12.0.0",
     "google-auth-library": "*",
-    "googleapis": "^126.0.1",
+    "googleapis": "*",
     "hnswlib-node": "^3.0.0",
     "html-to-text": "^9.0.5",
     "ignore": "^5.2.0",

